guide/inflation.rst

```diff
@@ -220,14 +222,23 @@ Using Relaxation to prior spread (RTPS; flavor **4**):
 ``&filter_nml :: inf_damping``
    *valid values:* 0.0 to 1.0
 
-   Applies to all inflation types, but most frequently used with time-adaptive inflation variants. DART defaults to
-   this damping method to apply inflation updates for regions of state space that are not influenced by observations
-   either because of gaps in availability or are out of range due to localization. To update the inflation, the
-   difference between the current inflation value and 1.0 is multiplied by this ``inf_damping`` value before the next assimilation
-   cycle. So the inflation values are pushed towards 1.0, from above or below (if ``inf_lower_bound`` allows inflation
-   values less than 1.0). A value of 0.0 turns all inflation off by forcing the inflation value to 1.0. A value of 1.0
-   turns damping off by leaving the original inflation value unchanged. We recommend setting the damping to a value
-   of 0.9, which damps slowly.
+   Inflation damping applies to all types of inflation but is most commonly used with time-adaptive inflation
+   methods. It is particularly useful especially in regions of state space that are not influenced by observations,
+   either due to gaps in data availability or because they fall outside the localization range. To damp the inflation,
+   the difference between the current inflation value and 1.0 is multiplied by this ``inf_damping`` value before
+   the next assimilation cycle. So the inflation values are pushed towards 1.0, from above or below
+   (if ``inf_lower_bound`` allows inflation values less than 1.0). A value of 0.0 turns all inflation off
+   by forcing the inflation value to 1.0. A value of 1.0 turns damping off by leaving the original inflation
+   value unchanged. We recommend setting the damping to a value of 0.9, which damps slowly.
+
+	 .. note::
+		 Order of (prior) inflation operations within DART:
+
+		 - Inflation is read from file or namelist
+		 - Inflation damping is applied: :math:`\sqrt{\lambda} \leftarrow 1 + \rho(\sqrt{\lambda} - 1),` where :math:`\rho` is the damping factor
+		 - Damped Inflation is applied to the prior ensemble
+		 - Inflation is updated with observations
+		 - Updated inflation values are written out
```

Comment:
I really like this order of operations. It emphasizes that the inflation damping and inflation estimation with observations are performed in series.  Could you maybe number the steps, because I think this would emphasize that the updated inflation values from step 5 (updated inflation is written out), are copied back into step 1 (Read in inflation from restart file).    Also instead of just saying Inflation is updated with observations in step 4, it would really help if it was stated Inflation is updated with observations **using inf_flavor algorithm**, or **using equation 1** which is defined at the beginning of the section.   Something that really distinguishes between the damping update versus the observation update.